plexapi/collection.py

```diff
@@ -226,6 +257,39 @@ def sortUpdate(self, sort=None):
             raise BadRequest('Unknown sort dir: %s. Options: %s' % (sort, list(sort_dict)))
         self.editAdvanced(collectionSort=key)
 
+    def visibleUpdate(self, library=None, home=None, shared=None):
```

Comment:
This method would be part of the `HubManage()` object, similar to `~plexapi.myplex.AccountOptOut()`.

https://github.com/pkkid/python-plexapi/blob/master/plexapi/myplex.py#L1354-L1386

```python
def _updateVisibility(self, recommended=None, home=None, shared=None):
     # See comments below

def promoteRecommended(self):
    self._updateVisibility(recommended=True)

def promoteHome(self):
    self._updateVisibility(home=True)

def promoteShared(self):
    self._updateVisibility(shared=True)
```